app.py

```diff
@@ -37,9 +37,15 @@ def startup_event():
     else:
         logger.info("Running on CPU")
 
+    # Text tokenization enabled by default
+    tokenize_text = True
+    transformers_tokenize_text = os.getenv("TRANSFORMERS_TOKENIZE_TEXT")
```

Comment:
I think the name `TRANSFORMERS_TOKENIZE_TEXT` is a bit generic here. What about `SKIP_SENTENCE_TOKENIZATION` or `DIRECT_TRANSFORMERS_TOKENIZE` to indicate you the vectorizer is no longer batching by sentences.

Reply:
changed to `T2V_TRANSFORMERS_DIRECT_TOKENIZE `

Reply:
Wouldn't this be better if it was part of the `moduleConfig` in the schema? Similar to pooling config etc? That would allow setting this per class and also changing it at runtime.

Reply:
We would have to then expand the API so that we would pass settings also in the payload

Reply:
That would be a bigger change, but I like the idea that the modules would have the option of controlling how the module behaves

Reply:
Will add this to the modules feature list